core/arch/arm/plat-stm32mp2/stm32_util.h

```diff
@@ -33,6 +33,7 @@ static inline void stm32mp_register_gpioz_pin_count(size_t count __unused) { }
  * Generic spinlock function that bypass spinlock if MMU is disabled or
  * lock is NULL.
  */
```

Comment:
can remove the above inline comment.
Actually I think you can remove all the above inline comments.

Reply:
Removed all inline comments